jwplatform/client.py

```diff
@@ -93,15 +100,29 @@ def request(self, method, path, body=None, headers=None, query_params=None):
 
         return self.raw_request(method=method, url=path, body=body, headers=headers)
 
+    def request_with_retry(self, method, path, body=None, headers=None, query_params=None,
+                           http_connection_retry_count=3):
+        retry_count = 0
+        for _ in range(http_connection_retry_count):
+            try:
+                response = self.request(method, path, body=body, headers=headers, query_params=query_params)
+                return response
+            except RemoteDisconnected as rd:
```

Comment:
I would recommend [StrictHTTPErrors from this library](https://github.com/polishmatt/neterr#stricthttperrors) if we want to be thorough about this. RemoteDisconnected is one of many things that could go wrong.